keystore/filesystem/filesystem_backup.go

```diff
@@ -134,15 +132,15 @@ func isHistoricalFilename(name string) bool {
 	return err == nil
 }
 
-func readFilesAsKeys(files []string, basePath string, encryptor keystore2.KeyEncryptor, storage Storage) ([]*keystore.Key, error) {
+func readFilesAsKeys(files []string, basePath string, encryptor keystore.KeyEncryptor, storage Storage) ([]*keystore.Key, error) {
 	output := make([]*keystore.Key, 0, len(files))
 	for _, f := range files {
 		content, err := storage.ReadFile(f)
 		if err != nil {
 			return nil, err
 		}
 		// remove absolute first part, leave only relative to path
-		relativeName := strings.Replace(f, basePath+"/", "", -1)
+		relativeName := strings.Replace(f, filepath.Join(basePath)+"/", "", -1)
```

Comment:
may be `filepath.Clean` will be more relevant and clear to use? As I see, .Join uses it under the hood